### PR TITLE
Update MAUI contributing guidelines, PR template, issue templates

### PR DESCRIPTION
**Pull request by rachelkang**
* Reflects MAUI instead of Xamarin.Forms
* Reflects initial MAUI handler property work currently in place

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/spec.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-labels: proposal-open, t/enhancement ➕
+labels: proposal-open, enhancement
 assignees: ''
 
 ---
 
 # [The feature] 
 
-Provide a concise description of the feature and the motivation for adding it to Xamarin.Forms
+Provide a concise description of the feature and the motivation for adding it to MAUI
```
Comment (jsuarezruiz) [inline .github/ISSUE_TEMPLATE/spec.md:12]: .NET MAUI

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

**Review by hartez — APPROVED** [review]

**Review by hartez — APPROVED** [review]
Just make the changes Javier mentioned and I think it looks good!

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -27,51 +27,18 @@ Read and follow our [Pull Request template](PULL_REQUEST_TEMPLATE.md).
...
+All pull requests need to be reviewed and tested by at least two members of the MAUI team. We do our best to review pull requests in a timely manner, but please be patient! Two reviewers will be assigned and will start the review process as soon as possible. If there are any changes requested, the contributor should make them at their earliest convenience or let the reviewers know that they are unable to make further contributions. If the pull request requires only minor changes, then someone else may pick it up and finish it. We will do our best to make sure that all credit is retained for contributors.
```
[the remark this answers is not in the document]
Comment (rachelkang) [reply .github/CONTRIBUTING.md]: fixed

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-Thank you for your interest in contributing to Xamarin.Forms! In this document, we'll outline what you need to know about contributing and how to get started.
+Thank you for your interest in contributing to Microsoft.Maui! In this document, we'll outline what you need to know about contributing and how to get started.
```
[the remark this answers is not in the document]
Comment (rachelkang) [reply .github/CONTRIBUTING.md]: thanks! fixed

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**